src/debugging/DockerDebugConfigurationProvider.ts

```diff
@@ -75,10 +70,15 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
                     }
                 }
 
-                if (debugConfiguration.type === undefined) {
-                    // If type is undefined, they may be doing F5 without creating any real launch.json, which won't work
-                    // VSCode subsequently will call provideDebugConfigurations which will show an error message
-                    return null;
+                if (debugConfiguration.type === undefined &&

```

Comment:
Should we look specifically for `undefined` or should any falsy value trigger this?

Reply:
that's a good question. we can do `!debugConfiguration.type`  and make it so if `type` is an empty string and it still triggers it? it just depends on how flexible we want the debug experience to be.

Reply:
@danegsta any thoughts on this btw?

Reply:
What's the actual value of debugConfiguration in the event there isn't an existing launch.json? Is it just `{}`?

Reply:
just double checked and yeah exactly, debugConfiguration would be `{}`